cli/cmd/migrate_test.go

```diff
@@ -8,21 +8,19 @@ import (
 )
 
 func TestMigrate(t *testing.T) {
-	t.Cleanup(func() {
-		os.RemoveAll("cloudquery.log")
```

Comment:
Meant to be removed? or `os.RemoveAll(logFileName)` I suppose?

Reply:
It's now written inside the temporary directory, so it should get removed along with the temporary directory at the end of the test.